pixi/src/checkAggregation/statusBanner.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default async function getStatusId(
+  recommendationsPromise,
+  pageExperiencePromise,
+  safeBrowsingPromise,
+  linterPromise,
+  mobileFriendlinessPromise
+) {
+  try {
+    const linter = await linterPromise;
+    if (!linter.isLoaded) {
+      return 'invalid-url';
+    }
+    if (!linter.isAmp) {
+      return 'no-amp';
```

Comment:
What do you think about putting these all in an enum? i.e. `Status.NO_AMP`

Reply:
Might give a better overview what states we have.
But as long as we use it only once and the logic does not get too big I think it's not that urgent.